apps/studio/components/interfaces/Organization/restriction.constants.tsx

```diff
@@ -0,0 +1,57 @@
+import Link from 'next/link'
+import React, { type ReactNode } from 'react'
+
+export const RESTRICTION_MESSAGES = {
+  GRACE_PERIOD: {
+    title: 'Organization plan has exceeded its quota',
+    description: (date: string, href: string): ReactNode => (
+      <>
+        You have been given a grace period until {date}. <Link href={href}>Review usage</Link>
+      </>
+    ),
+  },
+  GRACE_PERIOD_OVER: {
+    title: 'Grace period is over',
+    description: (href: string): ReactNode => (
+      <>
+        Your projects will not be able to serve requests when you use up your quota.{' '}
+        <Link href={href}>Review billing</Link>
+      </>
+    ),
+  },
+  RESTRICTED: {
+    title: 'Services restricted',
+    description: (href: string): ReactNode => (
+      <>
+        Your projects are unable to serve requests as your organization has used up its quota.{' '}
+        <Link href={href}>Resolve billing issues</Link>
+      </>
+    ),
+  },
+  OVERDUE_INVOICES: {
+    title: 'Outstanding invoices',
+    description: (href: string): ReactNode => (
+      <>
+        Please <Link href={href}>pay your invoices</Link> to avoid service disruption
+      </>
+    ),
+  },
+  OVERDUE_INVOICES_FROM_OTHER_ORGS: {
+    title: 'Outstanding invoices in other organization',
+    description: (href: string): ReactNode => (
+      <>
+        Please <Link href={href}>pay invoices</Link> for other organizations to avoid service
+        disruption
+      </>
+    ),
+  },
+  MISSING_BILLING_INFO: {
+    title: 'Missing billing information',
+    description: (href: string): ReactNode => (
+      <>
+        Please <Link href={href}>add a billing address</Link> to avoid restrictions. If you are a
+        registered business, please add a tax ID too
```

Comment:
just a thought but wondering if we should add a link for tax ID as well
although technically that's a secondary action, and perhaps we can reserve the "CTA" for just the primary

Reply:
No qualms about multiple inline links. The tax ID however is a field within that same billing address section, so I think it’s okay to miss:

<img width="2402" height="1242" alt="CleanShot 2025-12-29 at 13 50 16@2x" src="https://github.com/user-attachments/assets/a349b77b-7c8b-4191-a79f-5679f2bd6678" />